Package.swift

```diff
@@ -25,6 +29,7 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-log.git", .upToNextMajor(from: "1.4.2")),
         .package(url: "https://github.com/swift-server/swift-backtrace.git", .upToNextMajor(from: "1.2.3")),
         .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"),
+        .package(url: "https://github.com/jpsim/Yams.git", .upToNextMajor(from: "5.0.5")),
```

Comment:
generally speaking I rather keep dependencies to minimum. do we have to use yaml, or can we use JSON also?

Reply:
I think the plugin can not avoid generating YAML to let developer work with the generated deployment descriptor.

It took me a bit of time but I get rid of the Yams dependency. I added my own YAML parser (a0d5499c316208014141b6140d89de0822d8b560) modeled after Foundation's `JSONEncoder`

Would this be acceptable ?

Reply:
OK, done, I embark a custom YAML encoder, based on Swift's JSON encoder.